components/engine-javascript/src/main/java/org/eclipse/dirigible/components/engine/javascript/service/JavascriptService.java

```diff
@@ -103,6 +111,7 @@ public JavascriptSourceProvider getSourceProvider() {
 	 * @return the object
 	 */
 	public Object handleRequest(String projectName, String projectFilePath, String projectFilePathParam, Map<Object, Object> parameters, boolean debug) {
+		parameters.put("springAppContext", applicationContext);
```

Comment:
Remove this. Use the `SpringBeanProvider`

Reply:
Removed everything related to applicationContext and this springAppContext context variable and changed the js to call SpringBeanProvider.getBean()